protos/dcs/coalition/coalition.proto

```diff
@@ -23,6 +23,100 @@ service CoalitionService {
    */
   rpc GetMainReferencePoint(GetMainReferencePointRequest)
       returns (GetMainReferencePointResponse) {}
+
+  // https://wiki.hoggitworld.com/view/DCS_func_getPlayers
+  rpc GetPlayers(GetPlayersRequest) returns (GetPlayersResponse) {}
+}
+
+message AddGroupRequest {
+	dcs.common.Country country = 1;
+	dcs.common.GroupCategory category = 2;
+	oneof template {
+	  GroundGroupTemplate ground_template = 3;
+	  ShipGroupTemplate ship_template = 4;
+	  HelicopterGroupTemplate helicopter_template = 5;
+	  PlaneGroupTemplate plane_template = 6;
+	}
+
+	message GroundGroupTemplate {
+		optional uint32 group_id = 1;
```

Comment:
Do you know what happens if the ID is already taken?

Reply:
I *think* it replaces the existing object but not entirely sure.